server/src/test/java/io/unitycatalog/server/utils/PopulateTestDatabase.java

```diff
@@ -267,6 +267,62 @@ public static void main(String[] args) throws JsonProcessingException {
       tx.commit();
     }
 
+    // create external partitioned table
```

Comment:
For creating external table, you can just use tableOperations as it's already supported.

Reply:
Also, please put a more detailed comment.

Reply:
> just use tableOperations

May you explain it little more, please? I see that both implementation of `TableOperations` required running UC Server. But in the `PopulateTestDatabase.java` there is no db and running server. Thanks in advance!

P.S> I put a more detailed comment, please take a look is a new one enough? Tnx in advance!

Reply:
Sorry Since this is PopulateTestDatabase Class, you can create/get an instance of TableRepository (Similar Catalog and Schema Repository) and use that to create an external table (using a CreateTable Request). and Maybe simplify the creation of previously added external tables as well.

Reply:
When you instantiate the TableRepository it will connect to the default db on its own (via HibernateUtil).

Reply:
Ok, let me try to do it!

Reply:
@ravivj-db I rewrite one of tables to using `TableRepository`. Please, take a look and say what do you think about it, before I changed previously added tables, catalogs, schemas, etc. Because from my point of view the new code does not like like simplification... Tnx in advance!

Reply:
Have reviewed, besides minor comments overall looks ok. Also, let's just merge this one for now. We can change the way previously added EXTERNAL tables are being created in a separate PR.